app/components/configure-minted-asset/ConfigureMintedAsset.tsx

```
@@ -0,0 +1,180 @@
+                    {props.nft.supply.toString()}
+                  </span>
+                </p>
+              </div>
+            </TabPanel>
+
+            <TabPanel>
+              <VStack spacing={0} alignItems={'flex-start'}>
+                <ListClaimConditions
+                  processingClaimConditions={processingClaimConditions}
+                  nftContract={videoContract}
+                  nft={props.nft!}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)